src/cultionet/augment/augmentation.py

```diff
@@ -86,6 +89,9 @@ def augment_time(
     xseg_warped = tsaug_to_feature_stack(
         xseg_warped, nfeas, nrows, ncols
     ).clip(0, 1)
+    
+    # Ensure reshaping back to original values
+    # assert np.allclose(xseg_original, tsaug_to_feature_stack(xseg, nfeas, nrows, ncols))
```

Comment:
@MatthewPierson90 I think this fix was needed because this assertion test failed before this change.